src/utils/Api.js

```diff
@@ -0,0 +1,151 @@
+//import { link } from "fs";
+
+class Api {
+  constructor({ baseURL, headers }) {
+    //We are destructuring the arguement and then assigning those permaters as private fields
+    this._baseURL = baseURL;
+    this._headers = headers;
+  }
+
+  getAppInfo() {
+    //call getUserInfo it in this array
+    return Promise.all([this.getInitialCards(), this.getUserInfo()]);
+  }
+
+  //1. Loading user information from the server
+  //Method to get user information -  other methods for working with the API, like getuserInfo (different baseURL)
+  getUserInfo() {
+    const url = `${this._baseURL}/users/me`;
+    console.log("Fetching user info from:", url);
+
+    return fetch(`${this._baseURL}/users/me`, {
+      method: "GET",
+      headers: this._headers,
+    })
+      .then((res) => res.json())
+      .catch((err) => console.log("Error fetching user info:", err));
```

Comment:
You should **not** use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request, and this end is in `index.js` after all `then` blocks. And in real projects the `catch` blocks usually open error popups and so on.

Please, remove it from `Api` and add it in `index.js`.